### PR TITLE
Improve aggregated-api-endpoint

### PR DESCRIPTION
**Pull request by my-git9**
Improve aggregated-api-endpoint

/kind cleanup

### CODE REVIEW

File: docs/userguide/aggregated-api-endpoint.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ To quickly experience this feature, we experimented with karmada-apiserver certi
 
 ### Step1: Obtain the karmada-apiserver Certificate
 
-For karmada deployed using `hack/local-up-karmada.sh`, you can directly copy it from the `/root/.kube/` directory.
+For Karmada deployed using `hack/local-up-karmada.sh`, you can directly copy it from the `/root/.kube/` directory.
 
-```
+```shell
```
Comment (RainbowMango) [inline docs/userguide/aggregated-api-endpoint.md:17]: By the way, do you know what's the difference between `bash` and `shell`?

Comment (my-git9) [reply docs/userguide/aggregated-api-endpoint.md:17]: In github's markdown, the shell and bash seem to be the same in terms of highlighting: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L5973

Comment (RainbowMango) [reply docs/userguide/aggregated-api-endpoint.md:17]: Got it. Thanks.

**Pull request merged**